### PR TITLE
addig the translated messge to the ui

### PR DESCRIPTION
**Pull request by Dutervil**
@dmdesimone addig the translated messge to the ui. Have a look please

### CODE REVIEW

File: omod/src/main/webapp/fragments/hiv/programLocationWarning.gsp [hunk 1]
```diff
@@ -2,12 +2,11 @@
 <div>
 <div id="hiv-program-location-info" style="background-color: lightcoral; padding: 10px; border-radius: 5px;">
       <ul>
-          <li>Session Location: <b>${sessionLocation.name}</b></li>
           <% if (hivProgramLocation == null) { %>
           <li>No location found for this patient</li>
          <% } else { %>
             <% if (hivProgramLocation.name != sessionLocation.name) { %>
-             <li>You're trying to modify a patient of location <b>${hivProgramLocation.name}</b> who is not in your location</li>
+             <li>${ui.message("pihcode.alert") }</li>
```
Comment (mogoodrich) [inline omod/src/main/webapp/fragments/hiv/programLocationWarning.gsp:9]: A small typo, the other codes are called "pihcore." (named after the module) not "pihcode.", can we change to be consistent?  Thanks!

**Review by mogoodrich — CHANGES_REQUESTED** [review]
Thanks @Dutervil  see my comment!

**Comment by Dutervil** [comment]
@dmdesimone Have a look please

**Comment by Dutervil** [comment]
@cioan Can you test this out please?

**Comment by cioan** [comment]
Looks good to me, @Dutervil !

**Comment by Dutervil** [comment]
@cioan Can I merge?

**Pull request merged**